index.html

```diff
@@ -123,267 +123,277 @@ <h1>Чем мы занимаемся?</h1>
+                        <div class="fast-start__description-progress">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
                         </div>
-                        <div class="profession__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
+                        <div class="fast-start__description-progress">
+                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Netus eget velit quisque accumsan amet tortor. Velit, volutpat egestas fringilla mi porttitor tempus. Placerat dui.
                         </div>
                     </div>
-                </div>
             </div>
         </div>
-        <div class="partners wrapper">
-            <div class="partners__logo">
-                <img src="img/Ellipse%202%20(Stroke).png">
-            </div>
-            <div class="partners__header">
-                Партнеры - топовые бренды
-            </div>
-            <div class="partners__top-images">
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-            </div>
-            <div class="partners__lower-images">
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-                <div class="partners__images">
-                    <img src="img/msi.png">
-                </div>
-            </div>
-            <div class="partners__table">
-                <div class="partners__main-tableline">
-                </div>
-                <div class="partners__second-tablelines">
-                    <div class="partners__second-tableline">
+        <div class="wrapper">
+            <div class="profession">
+                <div class="profession__header">
+                    Получите профессию прямо сейчас
+                </div>
+                <div class="profession__row">
+                    <div class="profession__column>">
+                        <div class="profession__about">
+                            <div class="profession__icon">
+                                <img src="img/icon-1.png">
```

Comment:
`alt` attribute is mandatory for `img` tag. Use `alt=""` if you don't know what to write.